CI build failing on Node 4

Introduced by https://travis-ci.org/jfmengels/eslint-plugin-fp/jobs/167645907

```
$ npm test
> eslint-plugin-fp@2.2.0 test /home/travis/build/jfmengels/eslint-plugin-fp
> xo && nyc ava
Error: Failed to load plugin no-use-extend-native: Cannot find module 'eslint-plugin-no-use-extend-native'
Referenced from: 
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.load (/home/travis/build/jfmengels/eslint-plugin-fp/node_modules/eslint/lib/config/plugins.js:129:26)
    at Array.forEach (native)
    at Object.loadAll (/home/travis/build/jfmengels/eslint-plugin-fp/node_modules/eslint/lib/config/plugins.js:151:21)
    at load (/home/travis/build/jfmengels/eslint-plugin-fp/node_modules/eslint/lib/config/config-file.js:504:21)
    at /home/travis/build/jfmengels/eslint-plugin-fp/node_modules/eslint/lib/config/config-file.js:391:36
    at Array.reduceRight (native)
npm ERR! Test failed.  See above for more details.
```

```
$ npm test
> eslint-plugin-fp@2.2.0 test /home/travis/build/jfmengels/eslint-plugin-fp
> xo && nyc ava
Error: Failed to load plugin no-use-extend-native: Cannot find module 'eslint-plugin-no-use-extend-native'
Referenced from: 
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.load (/home/travis/build/jfmengels/eslint-plugin-fp/node_modules/eslint/lib/config/plugins.js:129:26)
    at Array.forEach (native)
    at Object.loadAll (/home/travis/build/jfmengels/eslint-plugin-fp/node_modules/eslint/lib/config/plugins.js:151:21)
    at load (/home/travis/build/jfmengels/eslint-plugin-fp/node_modules/eslint/lib/config/config-file.js:504:21)
    at /home/travis/build/jfmengels/eslint-plugin-fp/node_modules/eslint/lib/config/config-file.js:391:36
    at Array.reduceRight (native)
npm ERR! Test failed.  See above for more details.
```

--- a/package.json
+++ b/package.json
@@ -47,12 +47,11 @@
   },
   "devDependencies": {
     "ava": "^0.17.0",
-    "eslint": "^3.7.1",
     "eslint-ava-rule-tester": "^0.1.1",
     "inject-in-tag": "^1.1.1",
     "nyc": "^6.4.4",
     "pify": "^2.3.0",
-    "xo": "^0.17.0"
+    "xo": "^0.15.1"
   },
   "peerDependencies": {
     "eslint": ">=3"

